src/components/map/ha-locations-editor.ts

```diff
@@ -133,7 +133,7 @@ export class HaLocationsEditor extends LitElement {
         .layers=${this._getLayers(this._circles, this._locationMarkers)}
         .zoom=${this.zoom}
         .autoFit=${this.autoFit}
-        .darkMode=${this.darkMode}
+        ?darkMode=${this.darkMode}
```

Comment:
This change wasn't strictly required I think because it's already only a boolean, but no harm.

Reply:
Yeah I wasn't sure. Without the missing check you specified I feel like I cannot trust if a boolean property will actually contain a boolean, or if it might be passed undefined by an instancing module, and I didn't want to have to go audit the entire hierarchy of instancing to review the code. 

Is that correct analysis?

Reply:
Yep, that's basically it, but I did that yesterday.  The only time it gets passed here is in onboarding from a media query, which is a strict boolean.